qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -210,6 +210,11 @@ def __init__(
             auth_client = self._authenticate_ibm_quantum_account(self._client_params)
             # Update client parameters to use authenticated values.
             self._client_params.url = auth_client.current_service_urls()["services"]["runtime"]
+            if self._client_params.url == "https://api.de.quantum-computing.ibm.com/runtime":
+                warnings.warn(
+                    "Features in versions of qiskit-ibm-runtime greater than 0.13.0 may not "
```

Comment:
Late, but doesn't this technically include 0.13.0? (.e. 0.13.0 and above?

Reply:
Yeah, I'll update the wording in another PR
 `greater than and including 0.13.0...`